includes/class-monk-activator.php

```diff
@@ -37,11 +37,21 @@ public static function activate() {
 			$language = 'en_US';
 		}
 
+		$default_post_category = get_term( get_option( 'default_category' ) );
+		$uncategorized_term    = get_term_by( 'name', 'Uncategorized', 'category' );
```

Comment:
Use `ID` on `get_term_by` and check if `ID = 1` to ensure that is uncategorized.